dyndns/domain_setup.py

```diff
@@ -61,6 +64,8 @@ def main(domain, settings='settings.txt'):
         api_url=config.urlAPI
     ))
 
+    force = input("Should we override any existing A records in the DNS zone (y/n)?") == 'y'
```

Comment:
I think asking for it every time is not the best way to solve. Command line parameter would be more suited.